preupg/xmlgen/xml_utils.py

```diff
@@ -309,11 +306,11 @@ def create_xml_from_ini(self, main):
         }
         for key in self.keys:
             if 'check_script' not in key:
-                raise MissingTagsIniFileError
+                raise MissingTagsIniFileError(tags="check_script", ini_file=main)
```

Comment:
Yes, here and also in the _check_recommended_fields_ function (called three lines below). It's quite a hack (or let's say quick patching of whatever problem that you can see across the whole code of Preupgrade Assistant).
I fixed it in PR 230: https://github.com/upgrades-migrations/preupgrade-assistant/pull/230/files#diff-ae96a7b8baaa7ea98cc9b74552c59221L311